src/client/site/home.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import '../assets/home.css';
+import data from '../data/data.json';
+import Create from '../action/create';
+import CreateRow from '../action/createRow';
+import Read from '../action/read';
+import Table from './table';
+import Footer from './footer';
+
+class Home extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      data: data, 
```

Comment:
What's the difference between `data` and `root`?

Reply:
I want to use `root` to save the origin table because I use `data` to create a table every time when it finishes an action and some action like `read` while affect the table's length. 

Ex.: If I only get one row after `read` action activate, then my table data will have only one element.